python/Ganga/GPIDev/Lib/Registry/PrepRegistry.py

```diff
@@ -185,13 +185,13 @@ def lookup(self, sharedir, unprepare=False):
         The optional parameter 'unprepare=True' can be set to call the unprepare method 
         on the returned objects.
         """
-        from Ganga.GPI import jobs, box, tasks
+        from Ganga.Core.GangaRepository import getRegistryProxy
         objectlist = []
-        for thing in jobs.select():
+        for thing in getRegistryProxy('jobs').select():
```

Comment:
Again, do these need to be proxy-wrapped? We want to avoid using proxy-level objects in our code as much as possible as it increases the chance of accidentally infecting our code with more proxy-wrapped objects.